ember/app/templates/index.hbs

```diff
@@ -44,22 +44,22 @@
       </div>
     </div>
     {{/if}}
-    {{#if hasPopularRestaurants}}
```

Comment:
This should also not have been touched. This section should not be shown/rendered if there are no results.